modules/client/test/integration/client/methods.test.ts

```diff
@@ -366,41 +384,55 @@ describe("Client", () => {
           ),
         );
 
+        const mockedClient = mockedGraphqlRequest.getMockedInstance(
+          client.graphql.getClient(),
+        );
+        const subgraphDao: SubgraphDao = {
+          createdAt: Math.round(Date.now() / 1000).toString(),
+          id: TEST_DAO_ADDRESS,
+          subdomain: "test",
+          metadata: `ipfs://${IPFS_CID}`,
+          plugins: [{
+            id: TEST_MULTISIG_PLUGIN_ADDRESS,
+            __typename: SubgraphPluginTypeName.MULTISIG,
+          }],
+        };
+        mockedClient.request.mockResolvedValueOnce({
+          dao: subgraphDao,
+        });
+
         const dao = await client.methods.getDao(daoAddress);
```

Comment:
Same here, why not using the type cast?
Personally I prefer the typecast to make it not null because I only have to write it once but if you prefer `!` go for it, but lets try to do it the same way on all places

Reply:
typecasting should be avoided as much as possible. and here it is possible to avoid so I don't do it

Reply:
I mean both are assertions and should be avoided when possible imo, here it does not mater much because we can assume that the mock is going to return what we tell him to return. So feel free to leave it this way or change it to a `null` check